pkg/kube/client.go

```diff
@@ -68,7 +70,7 @@ type Client struct {
 	// Namespace allows to bypass the kubeconfig file for the choice of the namespace
 	Namespace string
 
-	kubeClient *kubernetes.Clientset
+	kubeClient kubernetes.Interface
```

Comment:
I think this would have be reworked to not change the Client struct as it maintainers guide says `Go interfaces in pkg/ MUST NOT be modified`. Is that right?